### PR TITLE
Cli tests

### PR DESCRIPTION
**Pull request by marcauberer**
Add unit tests for cli component

### CODE REVIEW

File: cli/preprocessor_test.go [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+// ------------------------------------- getCommentIdenFromLang -------------------------------------
+
+func TestGetComIdenFromLang_Success1(t *testing.T) {
+	// Manual input
+	langList := []string{"yaml", "yml", "python", "py", "docker", "dockerfile"}
```
Comment (Splines) [inline cli/preprocessor_test.go:13]: note that python has block identifiers

File: cli/util/file.go [hunk 2]
```diff
@@ -2,17 +2,16 @@ package util
 
 import (
 	"io/ioutil"
-	"log"
 	"os"
 )
 
 // GetFileContents reads a file and returns its content as a string
```
Comment (Splines) [inline cli/util/file.go:8]: Is there something like JavaDocs for Go?

File: cli/util/text.go [hunk 3]
```diff
@@ -2,6 +2,7 @@ package util
 
 import "encoding/json"
 
+// IsJson checks if the input string is a valid json structure
 func IsJson(s string) bool {
 	var js map[string]interface{}
 	return json.Unmarshal([]byte(s), &js) == nil
```
Comment (Splines) [inline cli/util/text.go:8]: ok 😅🧐

**Review by Splines — APPROVED** [review]
Nice tests 😊

File: cli/util/text.go (hunk 3 above)
Comment (marcauberer) [reply cli/util/text.go:8]: Not sure what bothers you here 😉

File: cli/util/file.go (hunk 2 above)
Comment (marcauberer) [reply cli/util/file.go:8]: Yes there is, although its not as good as JavaDoc. But Go has an useful feature which warns you about undocumented public functions out of the box when using Go. Due to that reason, all public functions have a GoDoc comment attached.

File: cli/util/text.go (hunk 3 above)
Comment (Splines) [reply cli/util/text.go:8]: nothing, it's just a bit cryptic to gather the meaning of these two lines ;) But you've documented it, so it's fine

File: cli/preprocessor_test.go (hunk 1 above)
Comment (marcauberer) [reply cli/preprocessor_test.go]: Fixed it.

**Pull request merged**